### PR TITLE
Row file added 

### PR DESCRIPTION
**Pull request by Aakarsh-Sathish**
Row file has been added. 
Ignored d2h,likes and like as mentioned

### CODE REVIEW

File: w2/row.py [hunk 1]
```diff
@@ -0,0 +1,7 @@
+class ROW:
+    def __init__(self, t):
+        self.cells = t
+
+    @classmethod
```
Comment (Justin091136) [inline w2/row.py]: I don't think we need to use @classmethod.

File: w2/row.py [hunk 2]
```diff
@@ -2,6 +2,5 @@ class ROW:
     def __init__(self, t):
         self.cells = t
 
-    @classmethod
     def new(cls, t):
         return cls(cells=t)
```
Comment (Justin091136) [inline w2/row.py]: I don't think we need to have a new() function. We can just use default initializer to create a new class instance.

Comment (Aakarsh-Sathish) [reply w2/row.py]: so the init is sufficient is what you are saying right?

Comment (Justin091136) [reply w2/row.py]: Yes, since we discuessed in discord that we're going to use NUM() instead of NUM.new().

**Comment by Justin091136** [comment]
Looks good to me.

**Pull request merged**